canvasapi/course.py

```diff
@@ -1698,6 +1698,33 @@ def get_quiz(self, quiz):
 
         return Quiz(self._requester, quiz_json)
 
+    def get_quiz_overrides(self, **kwargs):
+        """
+        Retrieve the actual due-at, unlock-at, \
```

Comment:
generally, `\` is not needed in docstrings. Sphinx is able to work with the line breaks. The main exception to this is when breaking mid-link, as we often do in the `:calls:` section